meta/style_guide.md

```diff
@@ -0,0 +1,102 @@
+# Style Guide
+
+What follows are style guide requirements for the rule book project. These are conventions we should attempt to follow, but they are not hard and fast rules like the [syntax document](syntax.md).
+
+## Base Style Guide
+
+Generally use the [APA Style Guide](https://apastyle.apa.org) as the base style guide.
+
+[https://apastyle.apa.org](https://apastyle.apa.org)
+
+However, this style guide is overridden by (at least) the following:
+
+    - Use integer numbers for all numbers
```

Comment:
**Alan Yam (via Email):** I noticed that you are using numbers instead of writing out the word, e.g. “every quiz question is 1 of a particular set….”.  Is there a reason for using “1” instead of “one”?

Reply:
This is purely a style choice, but one based on a desire for improved comprehension. Studies have shown using Arabic numerals instead of English names or Roman numerals improves readability.